addons/test_mail/models/test_mail_models.py

```diff
@@ -74,6 +74,9 @@ class MailTestTicket(models.Model):
     email_from = fields.Char(tracking=True)
     count = fields.Integer(default=1)
     datetime = fields.Datetime(default=fields.Datetime.now)
+    company_id = fields.Many2one('res.company')
+    company_currency = fields.Many2one("res.currency", string='Currency', related='company_id.currency_id', readonly=True, tracking=True)
```

Comment:
I think we could instead have a new model in test_mail_corner_case_models.py . For example we already have 'mail.test.track.compute' to test computed fields tracking. We could have mail.test.track.currency and have a specific unit test for tracking currencies. That way we don't impact anything else in test suite.